9.2/run-postgresql-master.sh

```diff
@@ -2,27 +2,16 @@
 
 set -eu
 
-source ${HOME}/common.sh
+source $HOME/common.sh
```

Comment:
Hmm, I noticed we use `${HOME}` in some places, `$HOME` in others -- while the curly braces are not needed in that particular case.
What's this good practice about?